### PR TITLE
Basic slack support

### PR DESCRIPTION
**Pull request by abtris**
Basic slack support, you can post message to Slack via BB

### CODE REVIEW

File: blackbelt/slack.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        if not token:
+            token = config['slack']['access_token']
+            slack = Slacker(token)
+            self.slack = slack
+        if not token:
+            raise ValueError("Can't do things with Slack without access token. Run bb init.")
+
+        self.token = token
+
+    def post_message(self, message, room='#sre'):
```
Comment (honzajavorek) [inline blackbelt/slack.py]: are you sure room for this stuff (deploying etc.) should be sre?

**Comment by honzajavorek** [comment]
Please address the comment about room/channel. Otherwise looks good.

File: blackbelt/slack.py (hunk 1 above)
Comment (abtris) [inline blackbelt/slack.py]: This is leftover, i change into engine-room back.

**Comment by Almad** [comment]
@abtris One thing that would be **blocking** for me: completely no documentation //cc @honzajavorek

**Comment by honzajavorek** [comment]
@Almad completely right! issued #62 myself and now I forget about keeping it up to date :-1:

**Pull request merged**